addons/core/package.json

```diff
@@ -45,17 +45,19 @@
     "intl-unofficial-duration-unit-format": "^3.1.0"
   },
   "devDependencies": {
-    "@babel/eslint-parser": "^7.18.2",
-    "@babel/plugin-proposal-decorators": "^7.18.6",
+    "@babel/eslint-parser": "^7.21.3",
+    "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.8.1",
+    "@ember/string": "^3.0.1",
+    "@ember/test-helpers": "^2.9.3",
     "@embroider/test-setup": "^2.1.1",
     "@faker-js/faker": "^8.0.2",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
     "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.0.1",
     "doctoc": "^2.2.0",
-    "ember-cli": "^4.4.1",
+    "ember-cli": "~4.12.2",
```

Comment:
Previous upgrades use `minor` didn't they? Why switch to exact?

Reply:
@ZedLi you are right, I think it was just a mishap on my behalf, we should be doing minor and point releases.  The cli for ember-upgrade replaces the pre-fix.  I will fix

Reply:
Within `addons/auth` which has been recently upgraded, the declaration of `ember-cli` is as follows: "ember-cli": "^4.12.2", It will be good to keep consistency.